editor/script_create_dialog.cpp

```diff
@@ -894,9 +894,9 @@ ScriptLanguage::ScriptTemplate ScriptCreateDialog::_parse_template(const ScriptL
 				if (line.begins_with("space-indent")) {
 					String indent_value = line.substr(17, -1).strip_edges();
 					if (indent_value.is_valid_int()) {
-						space_indent = "";
-						for (int i = 0; i < indent_value.to_int(); i++) {
-							space_indent += " ";
+						int indent_size = indent_value.to_int();
+						if (indent_size > 0) {
+							space_indent = String(" ").repeat(indent_size);
```

Comment:
This potentially changes the behavior, is it intended? If not there should `space_indent = ""` added back.

Reply:
The condition should be `if (indent_size >= 0)`, as `String(" ").repeat(0) == ""`